app/controllers/api/v1/merchants/merchant_invoices_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Merchants::MerchantInvoicesController < ApplicationController
+
+	def index
+		render json: Merchant.find(params[:id]).invoices
```

Comment:
I can do that. I'm making another PR right now for invoices. Ill make this change.
@tmikeschu